apps/mobile/src/components/widgets/tokens/token-widget.layout.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import { useTheme } from '@shopify/restyle';
+
+import { Box, SheetRef, Theme } from '@leather.io/ui/native';
+
+import { Widget } from '../widget';
+
+interface TokenWidgetProps {
+  balance?: React.ReactNode;
+  children: React.ReactNode;
+  header?: React.ReactNode;
+  sheetRef?: React.RefObject<SheetRef>;
+}
+
+export function TokenWidgetLayout({ children, header }: TokenWidgetProps) {
+  const theme = useTheme<Theme>();
```

Comment:
Would be great to wrap this somewhere so we can just import the hook without passing the generic

Reply:
I opened this issue to do that https://github.com/leather-io/issues/issues/323